web/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 ## 1.2.0-wip
 - Added missing `FileReader` event getters: `onAbort`, `onError`, `onLoad`,
   `onLoadStart`, `onProgress`.
-- Removed renames `UnderlyingSource` and `UnderlyingSourceBase` where the types 
+- Removed renames `UnderlyingSource` and `UnderlyingSourceBase` where the types
   don't exist.
+- Add variadic arguments.
```

Comment:
nit: `Fixed generation of variadic arguments to generate 4 optional parameters.`